bedrock/base/templates/includes/banners/vpn-coupon-variations/vpn-coupon-promo-1.html

```diff
@@ -0,0 +1,48 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% if LANG == 'de' %}
+  {% set banner_title = "Sicher durchs Netz" %}
+  {% set banner_subtitle = "Mit 20 % Rabatt aufs Jahresabo!" %}
+  {% set banner_text = "Nutze jetzt den code <span class='c-coupon-code'>VPN20</span> beim checkout" | safe %}
+  {% set link_text = "Jetzt sparen" %}
+{% elif LANG == 'fr' %}
+  {% set banner_title = "Voyagez en toute sécurité" %}
+  {% set banner_subtitle = "20% sur l’abonnement d’un an" %}
+  {% set banner_text = "Entrez le code <span class='c-coupon-code'>VPN20</span> au paiement" | safe %}
+  {% set link_text = "J’en profite" %}
+{% else %}
+  {% set banner_title = "Safe travels" %}
+  {% set banner_subtitle = "20% off yearly plan" %}
+  {% set banner_text = "Use code <span class='c-coupon-code'>VPN20</span> at checkout" | safe%}
+  {% set link_text = "Try now" %}
+{% endif %}
+
+<aside class="c-banner hide-from-legacy-ie" id="vpn-coupon-promo-banner">
+  <div class="c-banner-inner">
+    <div class="c-coupon-wrapper c-coupon">
+      <div class="c-coupon-title-wrapper">
+        <h2 class="c-coupon-title">
+          {{ banner_title|safe }}
+        </h2>
+      </div>
+      <div class="c-coupon-content-wrapper">
+        <h3 class="c-coupon-subtitle">
+          {{ banner_subtitle|safe }}
+        </h3>
+        <p class="c-coupon-text">
+          {{ banner_text|safe }}
```

Comment:
`banner_text` is the only one that includes HTML so it's the only one that needs to be marked `safe`, you can omit it from the others.